chamge/amqp-edge-backend.c

```diff
@@ -23,13 +34,23 @@ struct _ChamgeAmqpEdgeBackend
   amqp_connection_state_t amqp_conn;
   amqp_socket_t *amqp_socket;
 
-  gboolean activated;
+  ChamgeEdgeStatus state;
+  guint process_id;
+  gchar *uid;
```

Comment:
`uid` has already been defined in parent `ChamgeNode`.
To retrieve the value, we need to consider adding a virtual function / or a protected function to  `ChamgeNode`.